### DISCUSSION TITLE
The PKSimTemplate DBUser

### DISCUSSION THREAD

**Discussion by ktduliegmailcom** [Open-Systems-Pharmacology/Forum#1623] [Questions & Problems]
Hello,
Please, where can I find the link to download the  Template Database?

Here is the message I received: no template "expression profile available" in the gene profile database

![image](https://github.com/Open-Systems-Pharmacology/PK-Sim/assets/91042736/b308c4c9-390c-4e39-92a5-c7aefc573a22)

**Comment by msevestre** [comment]
Hi @ktduliegmailcom,

The Template database is the database where all the templates are saved. As the message says, there is no template in the database. Once you create some and save them to your database, you will be able to retrieve them again

Check the documentation online https://docs.open-systems-pharmacology.org/ for more info

**Comment by HenrikCordes** [comment]
Hi @ktduliegmailcom ,

if you want to add gene expression profiles to your simulations you first have to add the respective databases for a species:
https://docs.open-systems-pharmacology.org/working-with-pk-sim/pk-sim-documentation/pk-sim-options#template-database-path 

To add the data base of cause you first have to download them. See below:

You find the latest gene expression data bases for different species here:
https://github.com/Open-Systems-Pharmacology/Gene-Expression-Databases/releases/tag/v3.0.1 

Here you find the human data base delivered with the OSP:
https://github.com/Open-Systems-Pharmacology/Suite/releases/tag/v11.2 

Hope this helps!

Best
Henrik

**Reply by JanSchlender** [reply]
Any reason why the gene database V3.0.1 is not part of the release V11.1 but rather the human gene database V2? And is there a difference between the two databases for human data?

**Reply by Yuri05** [reply]
Underlying data sources are different. 
V 3.0 is not included because it was not fully validated yet.

**Reply by HenrikCordes** [reply]
@JanSchlender : The DBs have different sources and we did not make a comparision between those. 
@Yuri05: Is there a schema we could follow to validate the DB?  What can/should we do to do this? 

Would be happy to move this forward.
Best
Henrik

**Reply by StephanSchaller** [reply]
This was done for the evaluation in the past: https://pubmed.ncbi.nlm.nih.gov/22293118/

But we could establish a process...

**Reply by HenrikCordes** [reply]
Hi @StephanSchaller,
I understand the publication from Michaela shows the advantage of mechanistic ADME processes in PBPK models building of the scientific arguemnt of enzyme kinetics. 

My understanding of @Yuri05 comment and our previous discussion would be that we need a process / technical validation / documentation that the data provided by the expression DB within PK-Sim does not alter the source data / has different concent? 

As I understood, the source data would comprise the explicit expression values and the meta information / annotation (species homolog, alternative names). 

I'm not to familiar with the validation topic and would like to start a separate discussion to identify a suitable process for the validation. 
Also, I would like to discuss new features/update for PK-Sim to adequately represent the expression data (for example: % Homology to human column for animal genes/proteins, Distribution of expression values from multiple sources etc.)  

Best
Henrik
